svg/search.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024"><path d="M768 731.8L622.4 586.2c27-34.7 43.2-78.1 43.2-125.4 0-113.1-91.7-204.8-204.8-204.8S256 347.7 256 460.8s91.7 204.8 204.8 204.8c47.3 0 90.8-16.2 125.4-43.2L731.8 768l36.2-36.2zm-460.8-271c0-84.8 68.8-153.6 153.6-153.6S614.4 376 614.4 460.8s-68.8 153.6-153.6 153.6-153.6-68.8-153.6-153.6z"/></svg>
```

Comment:
I think it defaults to black, this difference is (again) a sketch vs illustrator thing